scripts/template/index.html

```diff
@@ -2,13 +2,12 @@
 <html lang="en-US">
     <head>
         <meta charset="UTF-8">
-        <meta http-equiv="refresh" content="0; url={URL}">
-        <script type="text/javascript">
-            window.location.href = "{URL}"
-        </script>
-        <title>Page Redirection</title>
+        <title>Skygear HTTP API Reference</title>
     </head>
     <body>
-        If you are not redirected automatically, follow this <a href='{URL}'>link to documentation</a>.
+        <ul>
+            <li><a href="{AUTH_URL}">Auth Gear</a></li>
```

Comment:
I think we can try using swagger-ui configuration to show both in same page.

ref: https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md

Reply:
It seems that a separate doc for each gear is better for the developer. If the developer wants to use Auth Gear, they do not need to see the doc of Asset Gear. It is distraction for them.